data/2mass_j12475047-0152142.json

```diff
@@ -1,7 +1,7 @@
 {
     "Sources": [
         {
-            "source": "2MASS J12475047-0152142\t",
+            "source": "2MASS J12475047-0152142",
```

Comment:
This may be another thing to test for. I do strip special characters when writing out the json files, but the DB accepts any string. Maybe we can write a test to check for \n, \t, \r and similar.

Reply:
I can open issues describing the need for these tests.